tests/test_hook_payload_adapter.py

```
@@ -178,6 +178,60 @@ def test_parse_hook_event_for_claude_user_prompt_expansion(self) -> None:
+        self.assertIsNotNone(event)
+        self.assertEqual(event.source, "copilot")
+        self.assertEqual(event.type, EventType.SESSION_END)
+        self.assertEqual(event.session_id, "cp-1")
+
+    def test_parse_hook_event_for_copilot_subagent_start(self) -> None:
+        """subagentStart maps to SESSION_START (new Copilot event, 2026-05-18 spec)."""
+        payload = {
+            "source_tool": "copilot",
+            "hook_event_name": "subagentStart",
+            "sessionId": "cp-2",
+            "cwd": "/tmp",
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)